resources/telemetry/charts/fluent-bit/templates/configmap-file.yaml

```diff
@@ -0,0 +1,25 @@
+{{- if (empty .Values.existingConfigMap) -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "fluent-bit.fullname" . }}-files
+  labels:
+    {{- include "fluent-bit.labels" . | nindent 4 }}
+data:
+  labelmap.json: |
```

Comment:
Let's indicate in the name that it's related to the Loki pipeline

Reply:
I saw the config.extraFiles value of the Fluent Bit chart. Maybe let's use that one for the labelmap, then we can avoid naming collisions with other files.